.circleci/config.yml

```diff
@@ -291,15 +304,48 @@ jobs:
             fi
 
       - store_artifacts:
-          path: /tmp/build/outputs
+          path: android/app/build/outputs
 
       - save_cache: *save-npm-cache-linux
 
-      - save_cache:
-          name: Save gradle cache
-          key: android-{{ checksum "android/build.gradle" }}-{{ checksum  "android/app/build.gradle" }}
+      - save_cache: *save-gradle-cache
+
+      - persist_to_workspace:
+          root: .
           paths:
-            - ~/.gradle
+            - android/fastlane/report.xml
+            - android/app/build/outputs
+
+  upload-google-play-alpha:
+    <<: *defaults
+    docker:
+      - image: circleci/android:api-28-node
+
+    steps:
+      - checkout
+
+      - attach_workspace:
+          at: android
+
+      - run:
+          name: Store the google service account key
+          command: echo "$FASTLANE_GOOGLE_SERVICE_ACCOUNT" | base64 --decode > service_account.json
+          working_directory: android
+
+      - run: *update-fastlane-android
+
+      - restore_cache: *restore-gradle-cache
+
+      - run:
+          name: Fastlane Play Store Upload
+          command: |
+            bundle exec fastlane android alpha
+          working_directory: android
+
+      - store_artifacts:
+          path: android/app/build/outputs
+
+      - save_cache: *save-gradle-cache
```

Comment:
I can't figure out the case that we need to save this cache. This seems not be used by this job.